packages/fluentui/docs/src/examples/components/Pill/Variations/PillExampleSizes.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { Pill, Flex, PillProps } from '@fluentui/react-northstar';
+
+const PillSizesExample = () => (
+  <Flex gap="gap.medium" column>
+    <Flex gap="gap.medium">
```

Comment:
```suggestion
    <Flex gap="gap.medium">
```

Please check this one, it seems that we will only one `Flex` there 😉

Reply:
done